web_src/js/features/citation.js

```diff
@@ -40,28 +40,35 @@ export async function initCitationFileCopyContent() {
     $citationCopyApa.toggleClass('primary', !isBibtex);
   };
 
-  try {
-    await initInputCitationValue($citationCopyApa, $citationCopyBibtex);
-  } catch (e) {
-    console.error(`initCitationFileCopyContent error: ${e}`, e);
-    return;
-  }
-  updateUi();
+  $('#cite-repo-button').on('click', async () => {
+    const moreBtn = document.getElementById('more-btn');
+    moreBtn.classList.add('is-loading');
```

Comment:
Why "more-btn"?

Reply:
Because the menu immediately closes on click so this button is the only remaining place where wec can show an indicator.

Reply:
I think `closet(".ui.dropdown")` is clearer for this case (and add a comment for it?)

Reply:
If we migrate this away from `.ui.dropdown`, it would break. I prefer depending on `id` attribute, which is more stable imho.

Reply:
But the `more-btn` is an abuse, and it is a bad name. I guess the `more-btn` will be removed ahead before "migrate this away from .ui.dropdown"

![image](https://github.com/go-gitea/gitea/assets/2114189/7cddd757-7fc8-4dba-b9d2-517e372c7e16)

Reply:
Done in https://github.com/go-gitea/gitea/pull/29855/commits/19d190822a8bbea087afb064a14db1adf5484e41.